src/traces/parcats/defaults.js

```diff
@@ -95,25 +95,18 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
     coerce('sortpaths');
     coerce('counts');
 
-    var labelfontDflt = {
-        weight: layout.font.weight,
-        style: layout.font.style,
-        variant: layout.font.variant,
-        family: layout.font.family,
-        size: Math.round(layout.font.size),
-        color: layout.font.color
-    };
-
-    Lib.coerceFont(coerce, 'labelfont', labelfontDflt);
-
-    var categoryfontDefault = {
-        weight: layout.font.weight,
-        style: layout.font.style,
-        variant: layout.font.variant,
-        family: layout.font.family,
-        size: Math.round(layout.font.size / 1.2),
-        color: layout.font.color
-    };
-
-    Lib.coerceFont(coerce, 'tickfont', categoryfontDefault);
+    var layoutFont = layout.font;
+
+    Lib.coerceFont(coerce, 'labelfont', layoutFont, {
+        overrideDflt: {
+            size: Math.round(layoutFont.size)
+        }
+    });
+
+    Lib.coerceFont(coerce, 'tickfont', layoutFont, {
+        autoShadowDflt: true,
```

Comment:
Do we need `autoShadowDflt` in both the `coerceFont` call and in the `attribute definition`? Can't we make it so the one in the attribute definition is enough?

Reply:
By looking into the `dlft` of the attribute?
I think that would become more complex.
This is just for few places e.g. in `parcats` and `parcoords` where we already have text shadows by default.

Reply:
I bet now that we have this we're going to add it to more things... and aside from being repetitive it just feels risky to require anyone adding this to know they need to add it in both places.

Reply:
I see your point there. But unfortunately changing this require notable changes to the way `coerceFont` is called or alternatively making an exception for `shadow` in `Lib.coerce`?
I'd be happy to give it a try in a separate PR as it could complicate this PR.

Reply:
Fair enough. Let’s then just make sure to address this the next time we add `autoShadowDflt` to another attribute.